docs/src/index.md

```diff
@@ -454,7 +699,7 @@ Some examples are shown below.
 While the semver specification says that all versions with a major version of 0 are incompatible with each other, we have made that choice that
 a version given as `0.a.b` is considered compatible with `0.a.c` if `a != 0` and  `c >= b`.
 
-#### Tilde specifiers
+##### Tilde specifiers
```

Comment:
Could there be an example of using Caret or Tilde specifiers in a `project.toml` to specify the dependency versions?